Shared/Samples/Show viewshed from geoelement in scene/ShowViewshedFromGeoelementInSceneView.swift

```diff
@@ -0,0 +1,220 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ShowViewshedFromGeoelementInSceneView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// The timer for the moving tank animation.
+    @State private var animationTimer: Timer!
+    
+    var body: some View {
```

Comment:
Please remove `animationTimer` as it is no longer being used.

```suggestion
    @StateObject private var model = Model()
    
    var body: some View {
```